dao/src/main/resources/db/migration/V90__scanSummaryHash.sql

```diff
@@ -0,0 +1,3 @@
+-- Add a hash column to the scan_summary table to make it easy to find duplicates.
+
+ALTER TABLE scan_summaries ADD COLUMN hash TEXT NOT NULL DEFAULT '';
```

Comment:
I don't understand, why does the hash value not have to be calculated for existing summaries?

Reply:
Explained below.